unimol_tools/README.md

```diff
@@ -18,6 +18,7 @@ wget https://github.com/dptech-corp/Uni-Mol/releases/download/v0.1/mol_pre_no_h_
 wget https://github.com/dptech-corp/Uni-Mol/releases/download/v0.1/pocket_pre_220816.pt
```

Comment:
you can align with 'poc_pre_220816.pt'

Reply:
I have made the required changes and successfully passed all the tests.